### ISSUE TITLE
fine tuning error

### ISSUE THREAD

**Issue by JosephChenHub** [antingshen/resnet-protofiles#4]
I'm using your prototxt to fine tune ResNet50 with the Kaiming He's caffemodel,but get an error:
 " Check failed: target_blobs.size() == source_layer.blobs_size() (1 vs. 2) Incompatible number of blobs for layer conv1 "

do you know how to solving it ?

**Comment by antingshen** [comment]
I don't think he released caffe prototxts for his model, so while I tried to make mine as close as possible, it may not be compatible with his weights. 

You can maybe try:
- Try to adjust this prototxt or his weights to fit each other by finding the difference (please let us know if you do)
- Train from scratch using this prototxt and then fine-tune

**Comment by JosephChenHub** [comment]
请看这个：https://zhuanlan.zhihu.com/p/22624331
“关于model，值得一提的是，官方的model只有conv1有bias_term,后面的那些卷积层均没有偏置项，所以在生成prototxt时，需要把那些层的bias_term禁用。(这个地方，当时被坑了很久，后来找了很长时间，终于找到了这个bug) ”

**Comment by antingshen** [comment]
I don't have a `bias_term` on any of the convolutions because a bias term should be redundant with batch normalization. Whether this creates an incompatibility with his model I haven't tried. Perhaps try enabling the bias term in `conv1` and see if that fixes your issue? 

I'd also be interested to know why there would need to be a bias term on `conv1`

**Comment by JosephChenHub** [comment]
it works well after adding a bias item to conv1 layer, however why only conv1 has bias is also pulzzing to me. Thanks a lot！

**Comment by antingshen** [comment]
Cool!
